packages/react-inbox/src/components/Messages/Header/index.tsx

```diff
@@ -45,7 +45,7 @@ const Header: React.FunctionComponent<IHeaderProps> = ({
                 {title}
                 {unreadMessageCount ? ` (${unreadMessageCount})` : ""}
               </h3>
-              <MessageSubHeader>Here is what you missed</MessageSubHeader>
+              {/*<MessageSubHeader>Here is what you missed</MessageSubHeader>*/}
```

Comment:
are we not ready to expose this change yet?

Reply:
we shouldn't expose it to yet no.  some people are trying to implement inbox right now and i don't think we are ready for them to see preferences yet right?